src/twisted/mail/pop3.py

```diff
@@ -565,7 +565,8 @@ def state_COMMAND(self, line):
             return self.processCommand(*line.split(' '))
         except (ValueError, AttributeError, POP3Error, TypeError) as e:
             log.err()
-            self.failResponse('bad protocol or server: %s: %s' % (e.__class__.__name__, e))
+            self.failResponse(
+                'bad protocol or server: %s: %s' % (e.__class__.__name__, e))
```

Comment:
If you want to change these lines, I'd suggest this style:

    self.failResponse(
                'bad protocol or server: %s: %s' % (e.__class__.__name__, e)
    )